scripts/build_npm.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  build,
+  emptyDir,
+  PackageJson,
+} from "https://deno.land/x/dnt@0.38.1/mod.ts";
+import { bold, cyan, green } from "$std/fmt/colors.ts";
+
+const OUT_DIR = "./dist";
+
+const version = Deno.args[0] || "beta";
```

Comment:
just tested it out and `beta` is not a valid value for the `package.version` field. We might wanna do just current + append `-beta`?

Reply:
Where did you run into issues with this? All the tests seem to pass fine at least...